site/content/docs/developer-guide/plugin-manifest.md

```diff
@@ -99,6 +100,8 @@ Krew makes it possible to install the same plugin on different operating systems
 (e.g., `windows`, `darwin` (macOS), and `linux`) and different architectures
 (e.g., `amd64`, `386`, `arm`, `arm64` and `ppc64le`).
 
+See [all supported platforms](https://github.com/kubernetes-sigs/krew/blob/master/cmd/validate-krew-manifest/main.go#L233-L249).
```

Comment:
This link will almost certainly go out of date. :)

I think the right approach is to point to krew repo's releases page and say "Only plugins in platforms that krew itself is distributed are supported.", right?